pkg/metrics/exporter.go

```diff
@@ -0,0 +1,24 @@
+package metrics
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/klog"
```

Comment:
Let's use `k8s.io/klog/v2` to be consistent with other packages

Reply:
fixed